src/mds/MDSRank.cc

```diff
@@ -2675,6 +2675,7 @@ void MDSRankDispatcher::handle_asok_command(
       f->close_section();
 
       bufferlist outbl;
+      f->flush(outbl); /* even for errors, dump f */
       do_respond(rc, {}, outbl);
```

Comment:
I just realized that it would be nice to pass the CSS here

```suggestion
do_respond(rc, css, outbl);
```

Reply:
done, PTAL

Reply:
❤️